dor/providers/process_basic_image.py

```diff
@@ -113,7 +115,9 @@ def get_file(self, function: list[UseFunction], format: UseFormat):
                     and format in result.file_info.uses
                 ):
                     return result
-        raise AccumulatorError("Result file not found")
+        raise AccumulatorError(
+            f"Result file not found for function {[str(use) for use in function]} and format {format}"
+        )
```

Comment:
@respinos and I talked about whether or not to raise an error here versus returning `None`. I plan to revisit this, but the change here at least gives better feedback -- mainly for development.